Makefile

```diff
@@ -0,0 +1,29 @@
+NAME    := miactl
+PACKAGE := github.com/mia-platform/$(NAME)
+GIT     := $(shell git rev-parse --short HEAD)
+SOURCE_DATE_EPOCH ?= $(shell date +%s)
+DATE    := $(shell date -u -d @${SOURCE_DATE_EPOCH} +%FT%T%Z)
```

Comment:
In osx this command produce a warning on `date` usage. It could be replaced with the following code, for example?
```sh
date -u -r ${SOURCE_DATE_EPOCH} +%FT%T%Z
```

Reply:
on linux you sample command does not work.
```
SOURCE_DATE_EPOCH=$(date +%s)
❯ date -u -r ${SOURCE_DATE_EPOCH} +%FT%T%Z
date: 1586798036: No such file or directory
```
Instead the makefile command works fine.
```
❯ date -u -d @${SOURCE_DATE_EPOCH} +%FT%T%Z
2020-04-13T17:13:56UTC
```
 I don't have a mac to try, what is the warning?

Reply:
It shows the usage info:
```sh
date -u -d @${SOURCE_DATE_EPOCH} +%FT%T%Z
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ...
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
```

Reply:
this is the simplest equivalent command I could think of, try it:
`TZ=UTC date +%FT%T%Z`

Reply:
This should be ok, also `date -u +%FT%T%Z` it's fine on macos

Reply:
fixed